jsonnet/components/node-exporter.libsonnet

```diff
@@ -290,6 +322,10 @@ function(params)
                           name: 'DBUS_SYSTEM_BUS_ADDRESS',
                           value: 'unix:path=/host/root/var/run/dbus/system_bus_socket',
                         },
+                        {
+                          name: 'ACCELERATORS_CONFIG_FILE',
```

Comment:
I don't think that we need to make it configurable given that it's a pure downstream feature: the accelerator collector config file location can be hardcoded in node_exporter and mounted by CMO at the expected path.

Reply:
Using the environment variable means that we determine the mounting path only at CMO, and there is no need to synchronize between CMO and node-exporter. IMHO in case it will be changed in the future for any reason, using ENV will allow us to avoid synchronization bugs

Reply:
I agree with Simon, hardcoding it would be better as we control the whole chain, and in case of a sync issue, we should be expecting the collector and the tests to yell.
(But if you think this is still needed I'd go for a cli arg for consistency, like `--collector.textfile.directory=/var/node_exporter/textfile` e.g.)

Reply:
i cannot add the cli arg in the current commit, since it will crush the node-exporter ( the collector.accelerators.configmpa-path flag is not defined there yet. So  in this PR i will just remove the EVN variable, and then add the flag in the node-exporter PR, and then will add another commit in this repo for defining the cli args